hub/core/version_control/commit_chunk_list.py

```diff
@@ -0,0 +1,30 @@
+from typing import List
+from hub.core.storage.cachable import Cachable
+
+
+class CommitChunkList(Cachable):
+    """Stores list of chunks stored for a particular tensor in a commit."""
+
+    def __init__(self) -> None:
+        self.chunks: List[str] = []
```

Comment:
hmm, for 1M chunks in a version, this would be appx. 4GB?

Reply:
no, it would be 9 MB